fmriprep/cli/parser.py

```diff
@@ -685,10 +704,9 @@ def parse_args(args=None, namespace=None):
         )
 
     # Setup directories
-    config.execution.log_dir = output_dir / "fmriprep" / "logs"
+    config.execution.log_dir = config.execution.fmriprep_dir / "logs"
     # Check and create output and working directories
     config.execution.log_dir.mkdir(exist_ok=True, parents=True)
-    output_dir.mkdir(exist_ok=True, parents=True)
```

Comment:
```suggestion
    config.execution.fmriprep_dir.mkdir(exist_ok=True, parents=True)
```
?

Reply:
`log_dir` is a subdirectory of `fmriprep_dir`.